src/Core/Scheduler/Providers/StaticProvider.js

```diff
@@ -128,20 +148,25 @@ export default {
             return false;
         }
 
-        for (const entry of layer.images) {
-            if (tile.extent.isInside(entry.extent)) {
-                return true;
+        if (tile.extent.crs() == layer.extent.crs()) {
```

Comment:
providing that Extent.isInside starts by doing `other.as(this._crs)`, why can't we use the spatial index in all cases by doing a `tile.extent.as(layer.extent.crs())` first?

Reply:
Ok, I'll change that.